lib/player.js

```diff
@@ -4308,6 +4308,19 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
               firstVariantWithLabel.language, '', 0, label);
 
       this.chooseVariantAndSwitch_(clearBuffer, safeMargin);
+    } else if (this.video_ && this.video_.audioTracks) {
+      const audioTracks = Array.from(this.video_.audioTracks);
+
+      let trackMatch = null;
+
+      for (const audioTrack of audioTracks) {
+        if (audioTrack.label == label) {
+          trackMatch = audioTrack;
+        }
+      }
+      if (trackMatch) {
+        this.switchHtml5Track_(trackMatch);
+      }
```

Comment:
Just curious: Is there any reason we use for/of over built in Array methods?
```ts
      const trackMatch = Array.from(this.video_.audioTracks)
                           .find(track => track.label == label);

      if (trackMatch) {
        this.switchHtml5Track_(trackMatch);
      }
```

Reply:
I have done it like this for consistency with the rest of the code

Reply:
@joeyparrish what do you think?

Reply:
The comment really was just a curiosity. I don't see anything wrong with this.

Reply:
I don't know if it matters or not.  I seem to recall that callback-based methods might have a performance penalty, maybe use more memory, I don't know.  We used to do a lot of array.forEach() when the codebase was all ES5, but there was some issue that caused us to go to for-of when we upgraded to ES6+.  I really can't recall what it was, and I could be misremembering the whole thing.  I won't make it a policy to use one style or the other without a very good reason, and I can't remember a reason.